### ISSUE TITLE
1.12.0-beta1 ctrl+backspace resolves incorrectly in combination with numlock/scrollock/capslock

### ISSUE THREAD

**Issue by GGAlanSmithee** [atom/atom#12980]
### Description

With `v1.12.0-beta1`, using a keyboard with standard swedish layout, it seems as if `ctrl+backspace` is bound to the wrong action, even if defined properly in the keymaps file.

(This worked in the latest stable release AFAIK.)

### Steps to Reproduce

1. Download latest beta releas https://github.com/atom/atom/releases/tag/v1.12.0-beta1
2. Toggle keybinder resolver `Ctrl+Period`
3. Type `Ctrl+backspace`
4. Observe that it says `Editor:delete-to-end-of-word`
5. Open the keymaps file by clicking the link in the keybinder resolver bar
6. Search for `ctrl-backspace`
7. Observe that it says `Editor:delete-to-beginning-of-word`

**Expected behavior:**

1. keymaps and keybinder resolver information matches
2. `ctrl+backspace` is bound to `Editor:delete-to-beggining-of-word`

**Actual behavior:**

1. keymaps and keybinder resolver information does not match
2. ctrl+backspace` is bound to `Editor:delete-to-end-of-word`

### Versions

```
$ atom --version
Atom    : 1.12.0-beta1
Electron: 1.3.6
Chrome  : 52.0.2743.82
Node    : 6.3.0

$ apm --version
apm  1.12.5
npm  3.10.5
node 4.4.5
python
git 2.8.1.windows.1
visual studio 2015
```

**Comment by Ben3eeE** [comment]
1. Can you reproduce this in safe mode `atom --safe`?
1. Have you checked the contents of your `keymap.cson`.

**Comment by Ben3eeE** [comment]
Also if you can upload a screenshot of your keybinding resolver that would be great.

**Comment by GGAlanSmithee** [comment]
@Ben3eeE 

1. Yes, it's reproducable in `atom --safe`
2. The keybinding resolver links to a `win32.cson` file in my AppData folder, cannot find a `keymap.cson`. The contents of the keymaps seems correct to me (see reproduction step 5-6-7)

> Also if you can upload a screenshot of your keybinding resolver that would be great.

This actually made me realize that the bug is with the keybinding resolver, because it resolves to `ctrl-delete` when I am hitting `ctrl-backspace`!

![keybindingresolver](https://cloud.githubusercontent.com/assets/11805103/19380397/6ccdf61c-91f7-11e6-9434-c417ba9c2de7.png)

**Comment by Ben3eeE** [comment]
Interesting, <kbd>Ctrl-Backspace</kbd> resolves correctly for me on Atom 1.12.0-beta1, Swedish layout, Windows 7.

Do you have any more information that can help reproduce this? Can you try the latest stable release and check if it resolves correctly there? Does <kbd>Backspace</kbd> without <kbd>Ctrl</kbd> resolve correctly? What about other modifier keys like <kbd>Alt</kbd> or <kbd>Shift</kbd>

**Comment by GGAlanSmithee** [comment]
* I am using Windows 10
* all keys with modifiers (including `ctrl`) seems to work, except `ctrl-backspace` which maps to `ctrl-delete`
* Installed `1.11.1` and it works as expected (`ctrl-backspace` is bound properly)

```
windows version 1607 (OS-version 14393.321)

$ atom --version
Atom    : 1.11.1
Electron: 0.37.8
Chrome  : 49.0.2623.75
Node    : 5.10.0

$ apm --version
apm  1.12.5
npm  3.10.5
node 4.4.5
python
git 2.8.1.windows.1
visual studio 2015
```

**Comment by Ben3eeE** [comment]
Thank you so much for reporting back, I have successfully reproduced this.
It seems to happen depend on the state of the <kbd>Num Lock</kbd>, <kbd>Caps Lock</kbd> and <kbd>Scroll Lock</kbd> keys.

I made a table of when it reproduces for me:

|Num lock|Caps lock|Scroll lock| <kbd>Ctrl-Backspace</kbd> Resolves| Result
|--------|----------|----------|---------------|---------
| On | On | On | <kbd>Ctrl-Backspace</kbd> | :+1:
| Off | Off | Off | <kbd>Ctrl-Delete</kbd>| :-1:
| On | On | Off | <kbd>Ctrl-Backspace</kbd>| :+1:
| On | Off | Off | <kbd>Ctrl-Backspace</kbd>| :+1:
| On | Off | On | <kbd>Ctrl-Backspace</kbd>| :+1:
| Off | On | On | <kbd>Ctrl-Backspace</kbd>| :+1:
| Off | Off | On | <kbd>Ctrl-Backspace</kbd>| :+1:
| Off | On | Off | <kbd>Ctrl-Delete</kbd>| :-1:

/cc: @nathansobo

**Comment by GGAlanSmithee** [comment]
@Ben3eeE thanks for your quick response and investigation, atom rocks

**Comment by nathansobo** [comment]
This should be fixed in 1.12.0-beta2, which was just published. Please let me know if you're still having problems with backspace.

**Comment by GGAlanSmithee** [comment]
> This should be fixed in 1.12.0-beta2, which was just published. Please let me know if you're still having problems with backspace.

I can confirm that it is working as intended. Thanks for fixing it so fast.
